pypechain/utilities/abi.py

```diff
@@ -672,14 +675,23 @@ def load_abi_infos_from_file(file_path: Path) -> list[AbiInfo]:
             bytecode = get_bytecode_from_json(json_file)
             # hardhat jsons have the contract name as a field.
             contract_name = json_file.get("contractName")
-            return [AbiInfo(abi=abi, bytecode=bytecode, contract_name=contract_name)]
+            # TODO
```

Comment:
whatcha wanna TODO?

Reply:
add TODOs to get link references for hardhat and solc as well.  should just work though i think